packages/frontend/src/helpers/isFieldHidden.ts

```diff
@@ -0,0 +1,23 @@
+import type { IField, IJSONObject } from '@plumber/types'
+
+import get from 'lodash.get'
+
+export function isFieldHidden(
+  hiddenIfCondition: IField['hiddenIf'] | null | undefined,
+  siblingParams: IJSONObject,
+): boolean {
+  if (!hiddenIfCondition) {
+    return false
+  }
+
+  const { fieldKey, fieldValue: expectedFieldValue, op } = hiddenIfCondition
+
+  switch (op) {
+    case 'always_true':
+      return true
+    case 'equals':
+      return expectedFieldValue === get(siblingParams, fieldKey)
+    case 'not_equals':
+      return expectedFieldValue !== get(siblingParams, fieldKey)
```

Comment:
noob question for my learning: do we leave out default for switch if there's a strict type check for the `op` already and thus, in this context, the default case will never be run?

Reply:
you're right, we don't need the default case.

I need to actually add the ESLint rule to ensure we have exhaustive switch checking...